helpers/helpers.go

```diff
@@ -61,6 +61,12 @@ var ProtoHelpersFuncMap = template.FuncMap{
 	"last": func(a []string) string {
 		return a[len(a)-1]
 	},
+	"index": func(a []string, index int32) string {
+		if len(a) > index {
+			return a[index]
```

Comment:
You should check the index is between  0 <= index < len(a)

Reply:
This will panic if index = -1

Reply:
Please add a example with this helper